src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -56,11 +56,14 @@ public function __construct(FactoryInterface $paymentMethodFactory, RepositoryIn
         $this->faker = \Faker\Factory::create();
         $this->optionsResolver =
             (new OptionsResolver())
+                ->setDefault('code', function (Options $options) {
+                    return StringInflector::nameToCode($options['name']);
+                })
                 ->setDefault('name', function (Options $options) {
                     return $this->faker->words(3, true);
                 })
-                ->setDefault('code', function (Options $options) {
-                    return StringInflector::nameToCode($options['name']);
```

Comment:
:-1: for moving this above name configuration, because we generate the code based on the name.